### ISSUE TITLE
Half of the bar chart has no data

### ISSUE THREAD

**Issue by CatCatcher1** [benjjneb/dada2#1918]
Hi, I used tutorial for my paired dataset until the last step. In last step, I got bar data for only 5 data in my dataset that contain 10 data. I will drop the picture of graph.
thanks for helping.

![image](https://github.com/benjjneb/dada2/assets/88461731/42a5c355-b6b6-4f6f-9444-302457ab606f)

**Comment by benjjneb** [comment]
I'm guessing the plot you are making is the bar plot at the end of the dada2 tutorial:

```
top20 <- names(sort(taxa_sums(ps), decreasing=TRUE))[1:20]
ps.top20 <- transform_sample_counts(ps, function(OTU) OTU/sum(OTU))
ps.top20 <- prune_taxa(top20, ps.top20)
plot_bar(ps.top20, x="Day", fill="Family") + facet_wrap(~When, scales="free_x")
```

If the second set of 5 samples do not have those top20 taxa in them, then this is what you will see. That is the most likely explanation for your results.

Given the difference in SRA numbers, it seems like it may be that the first 5 and the second 5 samples have no relation to each other, which would make this sort of thing more likely.

**Comment by CatCatcher1** [comment]
Yes I was using tutorial page, actually I am kinda new learner. My teacher give these SRA numbers as homework and wants to know taxonomic assignment and an object with relative abundance, taxonomy and metadata information of samples by using phyloseq and ggplot2. I got my samples and trimmed witth trimmomatic and paired merged-ends with FLASh. I followed the tutorial for single-end data and got the bottom. Only got that minor problem.

**Comment by benjjneb** [comment]
As per above I don't think anything has really gone wrong here. The issue is that there is little to no overlap between the first set of 5 samples and the second set of 5 samples in terms of the ASVs identified, which leads to the code above plotting only ASVs in the first 5 samples. Probably the first 5 are sequences more deeply than the second 5, so dominate the "top 20" most abundant taxa.
